dev_guide/secrets.adoc

```diff
@@ -181,7 +181,7 @@ but create new ones with distinct names.
 [[secrets-in-volumes-and-env-vars]]
 == Secrets in Volumes and Environment Variables
 
-See xref:secrets-examples[examples] of YAML files with secret data.
+If your `secret` uses an environment container, you must restart the container to see the updated `secret`. See xref:secrets-examples[examples] of YAML files with secret data.
```

Comment:
Not secret uses env container. Rather:
Should be: If your container uses secret as environment.
Or copy the k8s doc: If a container already consumes a Secret in an environment variable

Reply:
Going with the - ` Should be: If your container uses secret as environment.`